### ISSUE TITLE
Remove use of big-button class

### ISSUE THREAD

**Issue by bjgordon** [AusDTO/dta-website#70]
big-button class was copied from dto.gov.au, however it should be removed as the colours do not suit the rest of the site, may not be accessible contrast wise, and the uikit style should be used: https://gov-au-ui-kit.apps.staging.digital.gov.au/section-buttons.html
